src/Chessie/project.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.4.0",
+    "authors": ["Steffen Forkmann", "Max Malook", "Tomasz Heimowski"],
+    "description": "Railway-oriented programming for .NET",
+    
+    "compilationOptions": {
+    },
+
+    "compilerName": "fsc",
+    "compileFiles": [
+        "ErrorHandling.fs"
+    ],
+
+    "frameworks": {
+        "net46": {
```

Comment:
I use net46 as check for project.json before migrate libraries to .NET Core
If net46 works, dotnet cli works. 
After that, dnxcore errors are about migration full .net -> netcore ( or fsc bugs, but not that common )

Just pass `--framework dnxcore50` to `dotnet build`